tfhe/docs/references/fine-grained-apis/shortint/parameters.md

```diff
@@ -34,7 +34,7 @@ fn main() {
 
 ## Impact of parameters on the operations
 
-As shown [here](../../../getting\_started/benchmarks.md), the choice of the parameter set impacts the operations available and their efficiency.
+As shown [here](../../../getting_started/benchmarks/cpu_benchmarks.md), the choice of the parameter set impacts the operations available and their efficiency.
```

Comment:
this probably is not relevant anymore given we use a single parameter set for CPU benchmarks

Reply:
Could you suggest the wording change, please?